osu.Game/Screens/Ranking/Expanded/ExpandedPanelMiddleContent.cs

```diff
@@ -266,6 +270,9 @@ protected override void LoadComplete()
                         delay += 200;
                     }
                 }
+
+                if (!withFlair)
+                    FinishTransforms(true);
```

Comment:
This is again subjective but the D rank animation (the "broken rank" one, wherein it kinda rotates as if it was hanging off an anchor point and then stays crooked) looks a bit weird if it finishes immediately. Not sure whether not having it finish would be better or worse, though.

Reply:
Would be nice to have the little rotation part play separately, but in the interest of keeping the implementation simple, I think I'll leave it for now (unless you have a suggestion of how to make it work).